bioblend/galaxyclient.py

```diff
@@ -256,6 +260,18 @@ def make_patch_request(self, url, payload=None, params=None):
             status_code=r.status_code,
         )
 
+    def get_tus_uploader(self, path, url=None, storage=None, metadata=None, chunk_size=None):
```

Comment:
Is there an use case to have `url` as a parameter instead of hardcoded?

Reply:
Probably not although in theory you could have the tus endpoint be in a non-standard location.